contracts/test/lib.sol

```diff
@@ -0,0 +1,43 @@
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity 0.5.12;
+
+contract DSNote {
```

Comment:
Please rename the file to DSNote.sol?

Reply:
The file name is actually taken directly from the Maker repo: https://github.com/makerdao/dss/tree/master/src so might be better to keep as is?

Reply:
okay, makes sense. Don't worry, I will do some reorg of these later, possibly under a separate dss folder similar to compound contracts.